modules/oe_translation_cdt/tests/src/FunctionalJavascript/TranslationProviderTest.php

```diff
@@ -221,4 +304,24 @@ protected function assertLogMessagesTable(array $logs): void {
     $this->assertEquals($logs, $actual);
   }
 
```

Comment:
I am missing the cases where a language is cancelled. From the top of my head:
* One request with two languages: an accepted language and a cancelled language -> request finished
* One request with one language: cancelled language -> request finished.

Reply:
Done, I've also added a function that creates a new translation request. I'll move it to a trait when we have more functional tests.